cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -401,6 +408,52 @@ private static String _extractBasePath(String[] args) {
 		return defaultBasePath;
 	}
 
+	private static String _extractProfileName(String[] args) {
+		String defaultProfile = null;
```

Comment:
Here you set defaultProfile to null and later you return default profile.  But nowhere do you set defaultProfile to something other than null, so it isn't of much use.